docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -63,9 +63,11 @@ <h3>Applying the Stylesheet</h3>
             <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
         </tab>
         <tab heading="Non-Angular Sites">
-            <p>If your application does not use Node or Webpack, you can copy the UX Aspects stylesheet into your site's assets 
-                directory from <code>bower_components/ux-aspects/styles/ux-aspects.css</code> and include it in the head of your HTML.  
-                This is the only required step for non-Angular sites.</p>
+            <p>
+                If your application does not use Node or Webpack, you can apply the UX Aspects stylesheet from 
+                <code>bower_components/ux-aspects/styles/ux-aspects.css</code> by adding a <code>link</code> tag in the page header. 
+                This is the only required step for non-Angular sites.
+            </p>
```

Comment:
Do you think I should re-add the snippet for the link tag example?

Reply:
Might be good, that way it is a matter of someone simply copying and pasting rather than them having to come up with it themselves.

Reply:
good point.

Reply:
It's a bit odd to just assume that bower is used in that case. If you give something to copy and paste then brain tends to get disengaged.

Reply:
That's also a very good point. What should I do?

Reply:
Good point, my main issue was that simply stating to copy the stylesheet in would not be correct as assets would be missing.

Reply:
I believe that the important information to share is 1. where the CSS file can be found in the package; 2. that it depends upon the fonts and img directories.

Reply:
Yes agreed, I forgot to mention the dependencies.